addons/account_payment/views/account_portal_templates.xml

```diff
@@ -2,11 +2,33 @@
     <template id="portal_my_invoices_payment" name="Payment on My Invoices" inherit_id="account.portal_my_invoices">
         <xpath expr="//t[@t-call='portal.portal_table']/thead/tr/th[last()]" position="before">
             <th></th>
+            <th></th>
         </xpath>
         <xpath expr="//t[@t-foreach='invoices']/tr/td[last()]" position="before">
+            <t t-set="overdue_installments_amount" t-value="invoice.get_overdue_installments_amount()"/>
+            <t t-set="next_installment" t-value="invoice.get_next_installment_due()"/>
```

Comment:
Same here. Create a method python-side computing everything you need for the view